From: Guillaume Delacour <gui@iroqwa.org>
Description: Remove __{DATE,TIME}__ usage to do reproducible builds
 Suggestions come from upstream via irc
Forwarded: not-needed
Last-Update: 2015-02-12

Index: inspircd.new/src/inspircd.cpp
===================================================================
--- inspircd.new.orig/src/inspircd.cpp
+++ inspircd.new/src/inspircd.cpp
@@ -511,7 +511,7 @@ InspIRCd::InspIRCd(int argc, char** argv
 		}
 	}
 
-	std::cout << con_green << "Inspire Internet Relay Chat Server" << con_reset << ", compiled on " __DATE__ " at " __TIME__ << std::endl;
+	std::cout << con_green << "Inspire Internet Relay Chat Server" << con_reset << std::endl;
 	std::cout << con_green << "(C) InspIRCd Development Team." << con_reset << std::endl << std::endl;
 	std::cout << "Developers:" << std::endl;
 	std::cout << con_green << "\tBrain, FrostyCoolSlug, w00t, Om, Special, peavey" << std::endl;
Index: inspircd.new/src/users.cpp
===================================================================
--- inspircd.new.orig/src/users.cpp
+++ inspircd.new/src/users.cpp
@@ -785,7 +785,8 @@ void LocalUser::FullConnect()
 		this->WriteServ("NOTICE Auth :Welcome to \002%s\002!",ServerInstance->Config->Network.c_str());
 	this->WriteNumeric(RPL_WELCOME, "%s :Welcome to the %s IRC Network %s",this->nick.c_str(), ServerInstance->Config->Network.c_str(), GetFullRealHost().c_str());
 	this->WriteNumeric(RPL_YOURHOSTIS, "%s :Your host is %s, running version %s",this->nick.c_str(),ServerInstance->Config->ServerName.c_str(),BRANCH);
-	this->WriteNumeric(RPL_SERVERCREATED, "%s :This server was created %s %s", this->nick.c_str(), __TIME__, __DATE__);
+	// this->WriteNumeric(RPL_SERVERCREATED, ":This server was started %s", startup_timestr.c_str());
+	this->WriteNumeric(RPL_SERVERCREATED, "%s :This server was created on Debian", this->nick.c_str());
 
 	std::string umlist = ServerInstance->Modes->UserModeList();
 	std::string cmlist = ServerInstance->Modes->ChannelModeList();
